packages/frontapp/src/app/store/slices/userSlice.ts

```diff
@@ -29,7 +29,7 @@ const initialState: IUserState = {
   username: 'Test UserName',
   avatar: '',
   email: 'example@email.com',
-  userRole: RolesTypes.READER,
+  userRole: RolesTypes.MANAGER,
```

Comment:
i think `initialState` should be reader

Reply:
@sabrina34352 I think it's not important, because before signup/login we receive those data from our backend